### PR TITLE
update datasets class

### PR DESCRIPTION
**Pull request by boba-and-beer**

### CODE REVIEW

File: relevanceai/dataset_api/dataset.py [hunk 1]
```diff
@@ -58,7 +59,7 @@ def encode_documents(documents):
         self.client.pull_update_push(self.dataset_id, encode_documents)
 
 
-class Dataset:
+class Dataset(BatchAPIClient):
```
Comment (jtwinrelevanceai) [inline relevanceai/dataset_api/dataset.py:62]: Why does Dataset inherit from BatchAPIClient, when it has an internal variable self.client?

**Review by jtwinrelevanceai — COMMENTED** [review]
This looks good however I am confused where I've commented

File: relevanceai/dataset_api/dataset.py (hunk 1 above)
Comment (boba-and-beer) [reply relevanceai/dataset_api/dataset.py:62]: Removed :) thanks

**Review by jtwinrelevanceai — APPROVED** [review]
Looks good

**Pull request merged**